channels/ru.m3u

```diff
@@ -1,8 +1,13 @@
 #EXTM3U x-tvg-url="https://iptvx.one/epg/epg.xml.gz"
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Russian" tvg-logo="" group-title="",12 Канал (Омск)
 http://12channel.bonus-tv.ru:80/stream549837052987/tracks-v1a1/mono.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Russian" tvg-logo="" group-title="",1HD Music TV
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Russian" tvg-logo="" group-title="Music",1HD Music TV
 http://1hdru-hls-otcnet.cdnvideo.ru/onehdmusic/tracks-v1a1/index.m3u8
+
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Russian" tvg-logo="" group-title="Music",1MUS TV
+http://rtmp.one.by:1300/
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Russian" tvg-logo="" group-title="Music",1MUSICTV
+http://rtmp.one.by:2300/
```

Comment:
This channel is Belarusian and the links to it are already on the playlist:
https://github.com/iptv-org/iptv/blob/410294dc5191fa7902fb460bb2bb91a86c74c553/channels/by.m3u#L20-L27